### PR TITLE
Improved PR template (Issue 82)

### LINKED ISSUE #82: Update Contribution Guidelines/PR Template
- [ ] Remove issue link and, instead, clarify [this syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [ ] Add standards for PR subjects and commit messages
- [ ] Add standards for branch naming

### PR DESCRIPTION
**Pull request by R-Wang97**
Changelog:

- Updated PR template

Resolves #82

### CODE REVIEW

**Comment by DavidLu1997** [comment]
Can we put everything that isn't mandatory (e.g. instructions) in `[]`? And tell the submitter to delete them?

**Comment by R-Wang97** [comment]
ok

**Comment by Denton-L** [comment]
Branches should always be lowercase and hyphen-separated.

**Comment by R-Wang97** [comment]
ok

**Comment by Denton-L** [comment]
Also, I would suggest not naming the branch after the issue number because if we do that too much, it could make scrolling through the commit history a huge pain in the ass.

**Comment by R-Wang97** [comment]
Updated.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+**Changelog:**
 
-[Summary of changes that have been made in a list] 
+[*Summary of changes that have been made in a list format*]
 
-Ex: 
-1. [Changed...]
-2. [Implemented...]
-etc.
+1. [Fill in]
```
Comment (DavidLu1997) [inline .github/PULL_REQUEST_TEMPLATE.md]: Summary should really be `-`

**Comment by ian952** [comment]
👍

**Comment by DavidLu1997** [comment]
+1

**Pull request merged**